src/NetworkFunction/NetworkFunction.Autorest/custom/New-AzNetworkFunctionCollectorPolicy.ps1

```diff
@@ -168,22 +168,37 @@ param(
 
 process {
       $rg = $PSBoundParameters.ResourceGroupName
-
+      $hasIngestionPolicyIngestionSource = $PSBoundParameters.Remove('IngestionPolicyIngestionSource')
       # Ensure exr circuit bandwidth 1G or more
-      $cktname = $IngestionPolicyIngestionSource.ResourceId | Where {$IngestionPolicyIngestionSource.ResourceId -match "/*subscriptions/(?<subid>.*)/resourceGroups/(?<rgname>.*)/providers/Microsoft.Network/expressRouteCircuits/(?<circuitname>.*)"} | Foreach {$Matches['circuitname']}
-      Import-Module Az.Network -Force
-      $exrCircuit = Get-AzExpressRouteCircuit -Name $cktname -ResourceGroupName $rg
-      $bandwidthInGbps = $exrCircuit.BandwidthInGbps
-      $bandwidthInMbps = $exrCircuit.ServiceProviderProperties.BandwidthInMbps
-
-      if ($bandwidthInGbps -and ($bandwidthInGbps -lt 1)) {
-        throw "CollectorPolicy can not be updated because circuit has bandwidth less than 1G. Circuit size with a bandwidth of 1G or more is supported."
-      }
-
-      if ($bandwidthInMbps -and ($bandwidthInMbps -lt 1000)) {
-        throw "CollectorPolicy can not be updated because circuit has bandwidth less than 1G. Circuit size with a bandwidth of 1G or more is supported."
+      if ($hasIngestionPolicyIngestionSource) {
```

Comment:
Please add tests for this update.

Reply:
@YanaXu Existing tests will cover this. This is just a minor change in the way we parse resource id in the existing cmdlet.